fedimint-client/src/db.rs

```diff
@@ -296,6 +297,23 @@ impl_db_record!(
     db_prefix = DbKeyPrefix::ClientLastBackup
 );
 
+#[derive(Encodable, Decodable, Debug)]
+pub struct MetaFieldKey {
+    pub name: MetaFieldName,
+    pub priority: u64,
+}
+
+#[derive(Encodable, Decodable, Debug)]
+pub struct MetaFieldName(pub String);
+
+impl_db_record!(
+    key = MetaFieldKey,
+    value = String,
```

Comment:
I don't see `revision` being used anywhere. I specifically introduced monotonically incrementing revisions to make checking for updates more lightweight. Would be good to use it.

Reply:
will do in a follow-up, along with subscription for changes in meta. this is blocking some PR.

we don't promise database compat on master, so we won't need any migration

Reply:
doing in https://github.com/fedimint/fedimint/pull/4787

Reply:
added revision in this PR, because this doesn't have enough approvals yet :smile: